cms/utils/log_operations.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+from django.contrib.admin.models import LogEntry, ADDITION, CHANGE, DELETION
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import ugettext_lazy as _
+
+
+"""
+Logs are all collected here to keep the messages and contents consistent across all operations.
+May seem repetitive but allows each log to be unique for different circumstances.
+Method names make the log being created clear and concise
+
+Issues with current solution:
+    - A log entry isn't created when a page is created using the new page wizard. Again showing a flaw with testing via the API and not the actual fn's!!
+    - Difficult to find the log page creation in the current solution!! The existing implementation,uses the LogEntry model directly, others use the ModelAdmin Implementation. There should be one way where possible!!
```

Comment:
I think I'm getting confused about what "current solution" means :) I normally assume that current solution means whatever is in the code now (i.e. everything you added in this PR), but it sounds to me like you're actually talking about the previous solution here? Or I am veeery confused :)

Reply:
That's correct. The current solution is the logging that Paulo stated existed already i.e. page creation which isn't rue because the Wizard create a page doesn't currently generate a log.